src/parsec_client/wire_protocol.md

```diff
@@ -219,15 +228,17 @@ Field descriptions:
 
 - Magic number: a 32-bit unsigned integer whose value must be 0x5EC0A710 (selected to be an
    approximate transcoding of SECurity API). This field can be used as an initial validity check for
-   incoming messages. This field **must** be populated in outgoing messages.
+   incoming messages. This field **must** be populated in outgoing messages. This field will remain
+   the same across different wire protocol versions.
 - Header size: a 16-bit unsigned integer whose value is the size of the **remainder** of the header
    in bytes (once the magic number and header size fields have been consumed). Consumers **must**
    use this field to consume the correct number of bytes of header from the input stream, rather
-   than use this specification to deduce the header size.
+   than use this specification to deduce the header size. This field will remain the same across
```

Comment:
The last phrase sounds a bit confusing - i.e. sounds a bit like the value will stay the same, but I don't know how to fix that right now.

Reply:
Will modify to make it more clear.